docs/compatibility.md

```diff
@@ -17,6 +17,19 @@ Since there is no native token in Godwoken Polyjuice(while Ethereum has ETH as i
 
 - Use the `transfer function` in [CKB_ERC20_Proxy](https://github.com/nervosnetwork/godwoken-polyjuice/blob/3f1ad5b/solidity/erc20/README.md) contract [combined](https://github.com/nervosnetwork/godwoken-polyjuice/blob/3f1ad5b322/solidity/erc20/SudtERC20Proxy_UserDefinedDecimals.sol#L154) with sUDT_ID = 1 (CKB a.k.a. pETH).
 
+### Signing Transaction Only Support EIP155
+
+[EIP155](https://eips.ethereum.org/EIPS/eip-155) add the chainId for simple replay attack protection. currently we only support EIP155 signing scheme.
+
+### Result
```

Comment:
```suggestion
### Known issue #238
```

Reply:
issue 234 can not be click if change to this?